CHANGELOG.md

```diff
@@ -130,6 +130,7 @@ It is based on Tidepool Blip 1.27.
 - YLP-2506 Fix reservoir change icon always falling back to default
 - YLP-2510 Static dashboard does not perform calculation based on the last 14 days
 - YLP-2519 Fix refresh on Daily view disabling the forward arrow in the date picker
+- YLP-2523 Fix insulin values not rounded in Bolus tooltip
```

Comment:
You can also add `https://diabeloop.atlassian.net/browse/YLP-2496` in the bug fixed... We will have to write an E2E though :slightly_frowning_face: